pyiceberg/catalog/hive.py

```diff
@@ -276,7 +291,11 @@ def _create_hive_client(properties: Dict[str, str]) -> _HiveClient:
         last_exception = None
         for uri in properties["uri"].split(","):
             try:
-                return _HiveClient(uri, properties.get("ugi"))
+                return _HiveClient(
+                    uri,
+                    properties.get("ugi"),
+                    property_as_bool(properties, HIVE_KERBEROS_AUTH, HIVE_KERBEROS_AUTH_DEFAULT),
```

Comment:
this change is causing a few tests to fail, we need to add the extra arg to the tests
like so https://github.com/kevinjqliu/iceberg-python/pull/10/files#diff-8a0f847796be6745b3be158f5c39d0e83cbb868c162763a13895175b484cc529

Reply:
@kevinjqliu all green :)